examples/07-rule-chaining.js

```diff
@@ -83,17 +84,35 @@ engine
 facts = { accountId: 'washington', drinksOrangeJuice: true, enjoysVodka: true, isSociable: true }
 engine
   .run(facts) // first run, using washington's facts
+  .then((results) => {
+    // access whether washington is a screwdriverAficionado,
+    // which was determined at runtime via the rules `drinkRules`
+    return results.almanac.factValue('screwdriverAficionado')
+  })
+  .then(isScrewdriverAficionado => {
+    console.log(`${facts.accountId} ${isScrewdriverAficionado ? 'IS'.green : 'IS NOT'.red} a screwdriver aficionado`)
+  })
   .then(() => {
     facts = { accountId: 'jefferson', drinksOrangeJuice: true, enjoysVodka: false, isSociable: true }
     return engine.run(facts) // second run, using jefferson's facts; facts & evaluation are independent of the first run
   })
+  .then((results) => {
+    // access whether jefferson is a screwdriverAficionado,
+    // which was determined at runtime via the rules `drinkRules`
+    return results.almanac.factValue('screwdriverAficionado')
+  })
+  .then(isScrewdriverAficionado => {
+    console.log(`${facts.accountId} ${isScrewdriverAficionado ? 'IS'.green : 'IS NOT'.red} a screwdriver aficionado`)
+  })
```

Comment:
I went back and forth between extracting this out into a function, but ultimately decided to just duplicate the majority of this code to make it more explicit and not distract away from the important parts of this example.   Open to suggestions, though.

Reply:
excellent demonstration of how `almanac.getFact()` can be used post-runtime!